cmake/external/boringssl.cmake

```diff
@@ -22,10 +22,13 @@ endif()
 # the SOURCE_DIR when unpacking so this must come after grpc despite the fact
 # that grpc logically depends upon this.
 
-# grpc at v1.8.3 includes boringssl at be2ee342d3781ddb954f91f8a7e660c6f59e87e5
+# grpc v1.8.3 includes boringssl at be2ee342d3781ddb954f91f8a7e660c6f59e87e5
```

Comment:
Hmm, if we depend on `v1.24.3` now, it's not clear why the comment references `v1.8.3`, of all gRPC versions. Is this part of the comment still relevant? Perhaps consider either amending this paragraph slightly to make it clear why it's talking about `v1.8.3`:
```
# grpc v1.8.3, on which Firestore originally depended, includes boringssl...
```
or removing this paragraph and slightly rewording the next one:
```
# grpc v1.24.3 includes boringssl at b29b21a81b32ec273f118f589f46d56ad3332420
# (2018-05-22). While it does include gtest as a dependency, the version is woefully
# out of date.
```

Reply:
Done. You're right that the history isn't useful here. All that matters is that gRPC is woefully out of date so we're not following it exactly.